prow/workload-cluster/README.md

```diff
@@ -25,3 +25,24 @@ You can define two kinds of Secrets:
 The `secretspopulator` function looks for the `{prefix}.encrypted` object in the bucket and creates a Kubernetes Secret with a `{prefix}`.
 For service accounts, the Secret key is `service-account.json`. For generic Secrets, you must provide a key.
 For details on the file syntax, see the `RequiredSecretsData` type in [`secretspopulator`](../../development/tools/cmd/secretspopulator/main.go).
+
+## Configure a cluster to use Google Groups
```

Comment:
Cluster directory contains code related to prow instance and prow clusters, whereas this configuration is rather about permissions on test clusters.
As this is not description of any script or config file in our repo, the better place might be docs/prow?
My understanding is that, readme files outside docs/prow should describe content of directory and how to use it. Concepts and how to's for the whole repo should be placed in docs/prow

Reply:
I decided to create separate document fully dedicated to test clusters.